jest.config.js

```diff
@@ -1,5 +1,31 @@
+const esModules = [
```

Comment:
This is a giant list because `react-markdown` includes a lot of ESM dependencies that need to be transformed. I wanted to use the real implementation rather than our mocked one, so this replaces our library mock that we had before. This was also needed because `react-json-tree` was updated to ESM-only in the latest release.